include/solvers/navier_stokes_vof_assemblers.h

```diff
@@ -117,7 +117,7 @@ class GLSNavierStokesVOFAssemblerBDF : public NavierStokesAssemblerBase<dim>
  * @brief Class that assembles the Surface Tension Force (STF) for the
  * Navier-Stokes equations. The following equation is assembled
  *
- * $$\mathbf{F_{CSV}}=\sigma \rho k \nabla \phi
+ * $$\mathbf{F_{CSV}}=\sigma k \nabla \phi \frac{2 \rho}{\rho_1 + \rho_2}
```

Comment:
What is this rho here? Which rho is it, whereas rho_1 and rho_2 are for the phase? Maybe rho_0 and rho_1 then?

Reply:
Corrected. \rho is the density of the flow at each quadrature point.